docs/aci-container-features.md

```diff
@@ -0,0 +1,68 @@
+# Azure Container Instances: running single containerrs
+
+Single containes can be executed on ACI with the `docker run` command.
+A single container is executed in its own ACI container group, that will container only one container.
+
+Containers can be listed with the `docker ps` command, and stopped and removed with `docker stop <CONTAINER>` and `docker rm <CONTAINER>`.
+
+# Docker run options for ACI containers
+
+The table below lists supported `docker run` flags and their ACI counterparts.
+
+__Legend:__
+
+- __✓:__ Implemented
+- __n:__ Not yet implemented
+- __x:__ Not applicable / no available conversion
+
+| Flag                  |Map|  Notes
+|-----------------------|---|---------------------------------|
+| --cpus                | ✓ | See [Container Resources](#container-resources)
```

Comment:
Should be consistent and finish all sentences with `.`

Reply:
updated